packages/cloud-cognitive/src/components/Datagrid/Datagrid/Datagrid.js

```diff
@@ -41,6 +41,9 @@ export let Datagrid = React.forwardRef(({ datagridState, ...rest }, ref) => {
     verticalAlign = 'center',
     variableRowHeight,
     className,
+    gridTitle,
+    gridDescription,
+    DatagridDenseActions,
```

Comment:
Since the dense header is more of a variant, it may make more sense to create the variant as a boolean prop rather than another set of DatagridActions.

Reply:
@elycheea I'll make the changes requested and get back.

Reply:
@elycheea the consumer can now configure the dense header by setting the property in 'datagridState' as follows,

`const datagridState = useDatagrid({
    columns,
    data,
    gridTitle,
    gridDescription,
    initialState: {
      pageSize: 10,
      pageSizes: [5, 10, 25, 50],
    },
    useDenseHeader: true,
    DatagridActions,
    DatagridPagination,
  });`